src/index.ts

```diff
@@ -12,11 +12,12 @@ import {
 import {
   AUTHPATH,
   openAuthenticationWindow,
+  validateBaseURL,
```

Comment:
Can be used from `BaseConnector`?

Reply:
My BaseConnector doesn't seem to have this one. Maybe it's an older version but I don't want to update without testing again